spec/dummy/db/schema.rb

```diff
@@ -173,12 +173,13 @@
     t.datetime "updated_at", null: false
     t.integer "price_cents", default: 0, null: false
     t.string "price_currency", default: "'USD'::character varying", null: false
+    t.string "sizes", default: [], array: true
   end
 
   create_table "projects", force: :cascade do |t|
     t.string "name"
     t.string "status"
-    t.string "stage"
+    t.string "stage", default: [], array: true
```

Comment:
@zhephyn let's start by fixing the schema, the `AddSizesToProducts` migration is adding the `sizes` array column to the `products` table however the schema is also changing the `stage` column to array on the `projects` table.

```suggestion
    t.string "stage"
```

Reply:
I'm directly committing this change to speed up the process

Reply:
Okay no problem. Im not directly in contact with my computer at the moment

Reply:
@Paul-Bob . Though I didn't quite understand what you referenced, you specified a migration named AddSizesToProducts, yet according to the suggested change, it's like you're changing the stage column to a non array column, and the stage column is part of the projects table not the products table. Could you kindly clarify if this is the case?